drizzlepac/hlautils/se_source_generation.py

```diff
@@ -77,17 +77,20 @@ def create_sextractor_like_sourcelists(source_filename, catalog_filename, param_
 
     """
 
-    # Open the "white light" image and get the image data
+    # Open the "white light" image and get the SCI image data
     imghdu = fits.open(source_filename)
     imgarr = imghdu[1].data
```

Comment:
It would be more general to use '[('sci',1)]' instead of '[1]'.  In addition, if 'build' gets turned off, the while light image would be a simple FITS image, so adding logic to look in the PRIMARY header if 'sci,1' can not be found would guard against that situation.